api/mesh/v1alpha1/externalservice.proto

```diff
@@ -51,9 +51,12 @@ message ExternalService {
       // mTLS.
       google.protobuf.BoolValue allowRenegotiation = 5;
 
+      // If true then hostname verification will be skipped during certificate verification.
+      google.protobuf.BoolValue skipHostnameVerification = 6;
```

Comment:
In proto you shouldn't edit already existing fields. Just add it as 7th and that should be good 👍 https://protobuf.dev/programming-guides/proto2/#assigning